fdbclient/ServerKnobs.cpp

```diff
@@ -1143,6 +1143,7 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	// acceptable format: "<token_name1>$<absolute_file_path1>,<token_name2>$<absolute_file_path2>,.."
 	// NOTE: 'token-name" can NOT contain '$' character
 	init( REST_KMS_CONNECTOR_VALIDATION_TOKEN_DETAILS,             "");
+	init( ENABLE_REST_KMS_COMMUNICATION,                        false);
 	init( REST_KMS_CONNECTOR_REMOVE_TRAILING_NEWLINE,          false );
```

Comment:
nit: given you are here, please fix the spacing here as well :)

Reply:
I think we usually align this on the end bracket, so it should be fine?

Reply:
I meant the extra space between `false` and closing bracket :)